### PR TITLE
gh-107432 Fix incorrect indentation in annotations HOWTO

### PR DESCRIPTION
**Pull request by evildmp**
Body text in https://docs.python.org/3/howto/annotations.html was indented throughout, and was being rendered in blockquote elements.

* Issue: gh-107432

----
:books: Documentation preview :books:: https://cpython-previews--107445.org.readthedocs.build/

### CODE REVIEW

**Review by AlexWaygood — COMMENTED** [review]
Thanks! This looks good, and I think we should merge it. However, it's pretty unfortunate for `git blame` that this touches nearly every line in the file.

I wonder if we should add a `.git-blame-ignore-revs` file to CPython. After this is merged, perhaps we could add the commit to that file, so that digging through `git blame` is easier for future contributors. Curious if others have any thoughts on this.

**Comment by pradyunsg** [comment]
> Curious if others have any thoughts on this.

I think it's a good idea. This change is an improvement and, given that we have a mechanism to avoid the issues this could cause to git blame (which, IIUC, is the main issue this would cause), we should utilize that mechanism.

**Comment by AlexWaygood** [comment]
I've asked on the Core Dev discord if there are any objections to adding a `.git-blame-ignore-revs` file.

**Review by hugovk — APPROVED** [review]

**Comment by hugovk** [comment]
> I've asked on the Core Dev discord if there are any objections to adding a `.git-blame-ignore-revs` file.

There was support and no objections, and it should be mentioned in the devguide.

**Pull request merged**
